docs/components/movement-sensor/adxl345.md

```diff
@@ -1,135 +1,135 @@
-
-{{< readfile "/static/include/components/movement-sensor-control.md" >}}
+---
+title: "Configure an ADXL345 Accelerometer"
+linkTitle: "accel-adxl345"
+weight: 20
+type: "docs"
+description: "Configure an ADXL345 digital accelerometer."
+images: ["/icons/components/imu.svg"]
+# SMEs: Rand, Kim Mishra
+---
+
+The `accel-adxl345` movement sensor model supports the Analog Devices [ADXL345 digital accelerometer](https://www.analog.com/en/products/adxl345.html).
+This three axis accelerometer supplies linear acceleration data, supporting the `LinearAcceleration` method.
+
+If you are using a [Viam Rover](https://docs.viam.com/try-viam/), this is the accelerometer on it.
+
+{{< tabs >}}
+{{% tab name="Config Builder" %}}
+
+Navigate to the **Config** tab of your robot's page in [the Viam app](https://app.viam.com).
+Click on the **Components** subtab and click **Create component**.
+Select the `movement-sensor` type, then select the `accel-adxl345` model.
+Enter a name for your movement sensor and click **Create**.
+
+{{< imgproc src="/components/movement-sensor/adxl345-builder.png" alt="Creation of an `accel-adxl345` movement sensor in the Viam app config builder." resize="600x" >}}
+
+Edit and fill in the attributes as applicable.
+
+{{% /tab %}}
+{{% tab name="JSON Template" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "components": [
+    {
+      "name": "<your-sensor-name>",
+      "type": "movement_sensor",
+      "model": "accel-adxl345",
+      "attributes": {
+        "board": "<your-board-name>",
+        "i2c_bus": "<your-spi-bus-name-on-board>",
+        "use_alternate_i2c_address": <boolean>,
+        "tap": {
+           "accelerometer_pin": <int>,
+            "interrupt_pin": "<your-digital-interrupt-name-on-board>",
+            "exclude_x": <boolean>,
+            "exclude_y": <boolean>,
+            "exclude_z": <boolean>,
+            "threshold": <float>,
+            "dur_us": <float>
+        },
+        "free_fall": {
+            "accelerometer_pin": <int>,
+            "interrupt_pin": "<your-digital-interrupt-name-on-board>",
+            "threshold": <float>,
+            "time_ms": <float>
+        }
+      },
+      "depends_on": []
+    }
+  ]
+}
+```
+
+{{% /tab %}}
+{{% tab name="JSON Example" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "components": [
+    {
+      "name": "local",
+      "type": "board",
+      "model": "pi",
+      "attributes": {
+        "i2cs": [
+          {
+            "name": "default_i2c_bus",
+            "bus": "1"
+          }
+        ]
+      }
+    },
+    {
+      "name": "my-adxl",
+      "type": "movement_sensor",
+      "model": "accel-adxl345",
+      "attributes": {
+        "board": "local",
+        "i2c_bus": "default_i2c_bus",
+        "use_alternate_i2c_address": false
+      }
+    }
+  ]
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## Attributes
+
```

Comment:
Do you also want a `<!-- prettier-ignore -->` here too, and below?

Reply:
```suggestion

<!-- prettier-ignore -->
```